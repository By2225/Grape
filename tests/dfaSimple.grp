fun Int main() {
    Node<Bool> start = 'No';
    Node<Bool> oneA = 'No';
    Node<Bool> twoA = 'Yes';

    Graph<Bool, String> twoAs = <<
	start -"b"- start -"a"- oneA
	-"b"- oneA -"a"- twoA -"b"- twoA
	-"a"- 'No'
    >>;

    print(dfa(twoAs, start, "bbbbbabbb"));
}
