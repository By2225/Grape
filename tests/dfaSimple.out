./dist/grape: line 8: ./dist/grape.native: cannot execute binary file: Exec format error
./dist/stdlib.o: file not recognized: File format not recognized
collect2: error: ld returned 1 exit status
chmod: cannot access './out.g': No such file or directory
